src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -17,9 +26,14 @@ import {
 @Component({
   selector: 'sky-repeater',
   styleUrls: ['./repeater.component.scss'],
-  templateUrl: './repeater.component.html'
+  templateUrl: './repeater.component.html',
+  providers: [SkyRepeaterService]
```

Comment:
Just a question out of curiosity: will adding this existing service to the repeater component's providers mess up any of the logic that was already in the service (e.g, we're going from a module-provided service to a component-provided service)?

Reply:
This is a great question, and it struck me as strange we weren't already doing this before. Prior to my change, the service was only used for one thing: communicating a repeater item collapse from the item to the parent repeater component. In this prior state, if there were multiple repeaters on a single page, the service would transmit this change to ALL repeaters - that's definitely not the intent of the service. Those change emitters should be limited to the parent-child hierarchy from which they came from.

Also FWIW, switching to a component-level provider, we'll get better memory management - the service will be destroyed when the component is destroyed.